### DISCUSSION TITLE
Using dialog plugin to show custom component,why setting persistent to true don't work

### DISCUSSION THREAD

**Discussion by David-He** [quasarframework/quasar#9911] [General - Components / Directives / etc]
Using dialog plugin to show custom component,why setting persistent to true don't work?
code like:
onAddClick1:function(){
      this.$q.dialog({
        component: CustomComponent,
        
        
        persistent: true,
        // ...更多属性...
      }).onOk(() => {
        console.log('OK')
      }).onCancel(() => {
        console.log('Cancel')
      }).onDismiss(() => {
        console.log('Called on OK or Cancel')
      })
    },

**Comment by David-He** [comment]
OK, I know how to do it.
In custom component , add persistent property to q-dialog tag.

**Reply by ly2146** [reply]
Thank you for reading your comments
